CMSgov/easi-app
# Task List: Logic for Done vs Submitted states

# EASI-3110

## Changes and Description

Split out the old COMPLETED status for SUBMITTED or DONE in the task steps for Business Case Draft and Final.

## Testing
- Jest tests of the status states updated for Submitted vs Done
- Preview on local dev
  - Check out the preview branch `NOREF/gov-task-mock-preview` which pulls in mock data for all status states within the parent task list component
  - Turn on the `itGovV2Enabled` flag
  - Seed data or create a new IT Gov request to get to the task list view
